parley/src/layout/editor.rs

```diff
@@ -452,5 +486,7 @@ where
         self.layout.break_all_lines(self.width);
         self.layout.align(self.width, Alignment::Start);
         self.selection = self.selection.refresh(&self.layout);
+        self.dirty_layout = false;
+        self.generation += 1;
```

Comment:
```suggestion
        // Overflow handling: the generations should be compared only by value, so wrapping is fine.
        // This could break if the check happens to be performed exactly `2^32` generations later (2^64
        // on 64 bit), but that's sufficiently unlikely in practise.
        self.generation = self.generation.wrapping_add(1);
```
[`usize::wrapping_add`](https://doc.rust-lang.org/stable/std/primitive.usize.html#method.wrapping_add)